netlify.toml

```diff
@@ -1,3 +1,12 @@
 [[edge_functions]]
   function = "og-generate.png"
-  path = "/og-generate.png"
\ No newline at end of file
+  path = "/og-generate.png"
+
+# Specific branch deployment contexts
+#   Defining these ENV vars excludes certain docsets
+#   Use to have multiple separate netlify builds that don't time out
+[context.deploy-apidocs.environment]
```

Comment:
Note that this corresponds to the branch name that this PR is coming from (`deploy-apidocs`) and only applies to this specific branch deployment: https://deploy-apidocs--seqera-docs.netlify.app/